python/cuml/test/test_fil.py

```diff
@@ -572,11 +585,8 @@ def test_lightgbm(tmp_path, num_classes, n_categorical):
                                  n_estimators=num_round)
         lgm.fit(X_fit, y)
         lgm.booster_.save_model(model_path)
-        fm = ForestInference.load(model_path,
-                                  algo='TREE_REORG',
-                                  output_class=True,
-                                  model_type="lightgbm")
-        lgm_preds = lgm.predict(X)
+        lgm_preds = lgm.predict(X).astype(int)
```

Comment:
this line change improves error reporting for future test failures

Reply:
How does it improve error reporting?

Reply:
at a glance, I could tell whether the arrays are completely independent, somewhat similar, or seem almost identical.
Currently, comparing `[True, True, False, True, False, ...]` with `[1.0, 1.0, 0.0, 0.0, 1.0, ...]` takes extra time